inst/prelease.R

```diff
@@ -0,0 +1,115 @@
+# Open a pre-release issue on the dev_guide repo
+#
+# Uses the gh cli (https://cli.github.com), which must be installed and
+# authorised with 'gh auth'.
+#
+# The script calls the single function, `post_issue()`, defined at the end. All
```

Comment:
shouldn't there be other items based on the release guidance? I.e. items that remain the same for all releases

Reply:
oops, yes